diff --git a/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js b/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js
index 7dbddf5..94c74e6 100644
--- a/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js
+++ b/node_modules/@safe-global/safe-core-sdk-utils/dist/src/eip-3770/config.js
@@ -71,6 +71,7 @@ exports.networks = [
     { chainId: 2008, shortName: 'cloudwalk_testnet' },
     { chainId: 2019, shortName: 'pmint_test' },
     { chainId: 2020, shortName: 'pmint' },
+    { chainId: 2021, shortName: 'edg' },
     { chainId: 2221, shortName: 'tkava' },
     { chainId: 2222, shortName: 'kava' },
     { chainId: 3737, shortName: 'csb' },
